redom/android/src/main/java/com/pushtorefresh/redom/android/view/EditTextImpl.kt

```diff
@@ -0,0 +1,43 @@
+package com.pushtorefresh.redom.android.view
+
+import androidx.appcompat.widget.AppCompatEditText
+import com.jakewharton.rxbinding2.view.RxView
+import com.jakewharton.rxbinding2.widget.RxTextView
+import com.pushtorefresh.redom.api.Component
+import com.pushtorefresh.redom.api.EditText
+import com.pushtorefresh.redom.api.View
+import com.pushtorefresh.redom.api.toViewStructure
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+
+class EditTextImpl<O : Any> : EditText<O>, TextViewImpl<O>() {
+
+    override fun build(): Component<O, *> {
+        return EditTextComponent(
+                this,
+                EditText::class.java,
+                Observable.merge(outputObservables)
+        )
+    }
+
+    private class EditTextComponent<O : Any>(
+            private val editTextImpl: EditTextImpl<O>,
+            override val clazz: Class<out View<*>>,
+            override val output: Observable<O>
+    ) : Component<O, AppCompatEditText> {
+
+        override val viewStructure = toViewStructure(this)
+
+        override fun bind(view: AppCompatEditText): Disposable {
+            val disposable = CompositeDisposable()
+
+            editTextImpl.observeClicks?.also { RxView.clicks(view).subscribe(it) }
```

Comment:
you forget to add disposables to composite disposable.

Reply:
ouch